conversion.js

```diff
@@ -0,0 +1,30 @@
+var pry = require('pryjs')
+
+function convertToOldRoman(number) {
+  toPrint = ""
+  var arr = ('' + number).split('')
+  length = arr.length
+
+  arr.forEach (function (num) {
+    var zeroes = length - (arr.indexOf(num) + 1)
+    if (zeroes == 3) {
+      toPrint += "M".repeat(num);
+      eval(pry.it);
+    } else if ((zeroes == 2) && (num % 500 == 0)) {
```

Comment:
you're doing a lot of repeated logic here. i encourage you to get it to work using this logic, then think about how to extract some of this out!